### PR TITLE
Feature/fgcd 001

### PR DESCRIPTION
**Pull request by MatthewWaanders**

### CODE REVIEW

File: drivers/FGCD-001/driver.settings.compose.json [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        },
+        "value": 2
+      }
+    ]
+  },
+  {
+    "type": "group",
+    "label": {
+      "en": "Temperature sensor",
+      "nl": "Temperatuurssensor"
```
Comment (RobinBol) [inline drivers/FGCD-001/driver.settings.compose.json]: Temperatuursensor*

File: drivers/FGCD-001/driver.settings.compose.json [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+          "max": 85
+        },
+        "value": 55
+      }
+    ]
+  },
+  {
+    "type": "group",
+    "label": {
+      "en": "LED and acoustic notifications",
```
Comment (RobinBol) [inline drivers/FGCD-001/driver.settings.compose.json]: Visual/Visuele* to be consistent with your other drivers?

File: drivers/FGCD-001/driver.settings.compose.json [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+            "label": {
+              "en": "No Notifications",
+              "nl": "Geen Meldingen"
+            }
+          },
+          {
+            "id": "1",
+            "label": {
+              "en": "Casing Opened",
+              "nl": "Behuizing Geopend"
```
Comment (RobinBol) [inline drivers/FGCD-001/driver.settings.compose.json]: Here you use capitalisation, however, "Temperature too high" is not title cased.

File: drivers/FGCD-001/driver.settings.compose.json [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+            }
+          }
+        ],
+        "value": "0"
+      },
+      {
+        "id": "acoustic_notification",
+        "type": "dropdown",
+        "label": {
+          "en": "Acoutic Notification",
```
Comment (RobinBol) [inline drivers/FGCD-001/driver.settings.compose.json]: Be consistent with capitalisation, why does this setting have title case whereas most others don't?

File: drivers/FGCD-001/driver.settings.compose.json [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+            "id": "1",
+            "label": {
+              "en": "Casing Opened",
+              "nl": "Behuizing Geopend"
+            }
+          },
+          {
+            "id": "2",
+            "label": {
+              "en": "Temperature too high",
```
Comment (RobinBol) [inline drivers/FGCD-001/driver.settings.compose.json]: Same here

Comment (MatthewWaanders) [reply drivers/FGCD-001/driver.settings.compose.json]: Is fixed in the lines above where I changed title case to normal case.

**Pull request merged**